taskvine/src/manager/vine_manager.c

```diff
@@ -884,6 +1058,7 @@ void vine_manager_remove_worker(
 
 	hash_table_remove(q->worker_table, w->hashkey);
 	hash_table_remove(q->workers_with_available_results, w->hashkey);
```

Comment:
Now `workers_with_available_results` really means specifically `workers_with_watched_file_updates` so please call it that throughout.